uint8 state

uint8 INIT = 0                      
uint8 WAIT_TAKE_OFF_FINISHED = 1            
uint8 FLY_TO_STRAT = 2               
uint8 SEARCHING_PILLAR = 3             
uint8 ROTATE_PILLARS = 4       
uint8 FLYING_VIA_TUNNEL = 5            
uint8 SEARCHING_MAZE = 6       
uint8 PASSING_CIRCLES = 7            
uint8 SEARCHING_APRILTAG = 8            
uint8 LANDING = 9                      